data/50-org.ayatana.indicator.a11y.AccountsService.pkla

```diff
@@ -0,0 +1,6 @@
+[Allow LightDM to set AccountsService fields]
```

Comment:
@tari01 you are aware that .pkla files are deprecated in policykit? We need to move forward here across all indicators, probably.

Reply:
But not in this PR.

Reply:
Yes, there is an issue opened for this already. I keep skipping it, but will eventually get there.